include/boost/fusion/adapted.hpp

```diff
@@ -23,4 +23,12 @@
 #include <boost/fusion/adapted/std_tuple.hpp>
 #endif
 
+#if BOOST_FUSION_PFR_ENABLED
+#if BOOST_FUSION_PFR_ENABLE_IMPLICIT_REFLECTION
+#include <boost/fusion/adapted/boost_pfr.hpp>
```

Comment:
Add a regression test for it. Maybe you just need to include `adapted.hpp` in some old tests?

Reply:
Implemented regressiong tests. It's provided by `with_or_without_fallback.hpp` inclusion in every old test